.github/workflows/test-all-packages.yml

```diff
@@ -483,7 +488,9 @@ jobs:
       # END-TEST-BOILERPLATE
       - name: yarn test (boot)
         if: (success() || failure())
-        run: cd packages/boot && yarn ${{ steps.vars.outputs.test }} | $TEST_COLLECT
+        run: cd packages/boot && yarn ${{ steps.vars.outputs.test }} "test/**/${TESTGLOB}.test.*" | $TEST_COLLECT
```

Comment:
Does Ava error if you omit `.test` ?  Ideally the job doesn't have to know about the file naming pattern

Reply:
Addressed by adopting your advice re AVA's test splitting.